rasbet/models/models.py

```diff
@@ -0,0 +1,92 @@
+from django.db import models
+
+# Here we should type our models
+
+
+class Game(models.Model):
+    name = models.CharField(
+        max_length=100, primary_key=True)  # name of the game used as primary key
+    homeTeam = models.CharField(max_length=100)  # name of the home team
+    awayTeam = models.CharField(max_length=100)  # name of the away team
+    commanceTime = models.DateTimeField()  # time of the game
+    completed = models.BooleanField(default=False)  # is the game completed
+    # scores of the game in the format "NxN" where N is a number
+    scores = models.CharField(max_length=100, default=None, null=True)
+
+    def __str__(self):
+        return self.homeTeam + " " + self.awayTeam + " " + str(self.commanceTime) + " " + str(self.completed) + " " + str(self.scores)
+
+
+class User(models.Model):
+    username = models.CharField(
+        max_length=100, unique=True)  # username of the user
+    password = models.CharField(max_length=100)  # password of the user
+    email = models.CharField(
+        max_length=100, default=None, null=True, unique=True)  # email of the user
+    firstName = models.CharField(
+        max_length=100, default=None, null=True)  # first name of the user
+    lastName = models.CharField(
+        max_length=100, default=None, null=True)  # last name of the user
+    balance = models.FloatField(default=0.0, null=True)  # balance of the user
+    # user, specialist or admin
+    type = models.CharField(max_length=100, default="user")
+
+    def __str__(self):
+        return self.username + " " + self.email + " " + self.firstName + " " + self.lastName + " " + str(self.balance) + " " + self.type
```

Comment:
We can use django user model. Created PR [here](https://github.com/Suissinho2511/RASBET/pull/3).